ioctl/newcmd/config/config.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2022 IoTeX Foundation
```

Comment:
these 4 new files file another PR like "add config command"

Reply:
yes,  just add function, but no specific implementation. this will be done after merged

Reply:
delete